package/gluon-respondd/files/lib/gluon/respondd/neighbours.cache

```diff
@@ -0,0 +1 @@
+10000
```

Comment:
Didn't [you say](https://github.com/freifunk-gluon/packages/pull/125#discussion_r49883003) you want to change neighbours and statistics cache time to 900ms?